src/icons/checkbox-checked.svg

```diff
@@ -1,4 +1,4 @@
 <svg viewBox="0 0 20 20" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
I think we should make the rounded background with the use of CSS.

Reply:
done